pkg/build/strategies/sti/postexecutorstep.go

```diff
@@ -0,0 +1,459 @@
+package sti
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"syscall"
+
+	"github.com/openshift/source-to-image/pkg/api"
+	dockerpkg "github.com/openshift/source-to-image/pkg/docker"
+	"github.com/openshift/source-to-image/pkg/errors"
+	"github.com/openshift/source-to-image/pkg/tar"
+	"github.com/openshift/source-to-image/pkg/util"
+)
+
+type postExecutorStepContext struct {
+	// id of the previous image that we're holding because after committing the image, we'll lose it.
+	// Used only when build is incremental and RemovePreviousImage setting is enabled.
+	// See also: rememberPreviousImageStep and removePreviousImageStep
+	previousImageID string
+
+	// Container id that will be committed.
+	// See also: commitBuilderImageStep/commitRuntimeImageStep
+	containerID string
+
+	// See also: commitBuilderImageStep/commitRuntimeImageStep
```

Comment:
Yes, it is. It also the value of `io.openshift.s2i.destination` label.

>isn't it basically the working directory for assembly?

I'm not sure, maybe it is.